contrib/linux/dosbox-staging.metainfo.xml

```diff
@@ -9,16 +8,24 @@
   </provides>
   <content_rating type="oars-1.1" />
   <name>DOSBox Staging</name>
-  <summary>DOSBox Staging is modernized DOSBox soft-fork</summary>
+  <summary>DOS/x86 emulator focusing on ease of use</summary>
   <description>
     <p>
-      DOSBox Staging is a DOS-emulator using SDL 2 for easy portability to different platforms.
+      DOSBox Staging is a full x86 CPU emulator (independent of host architecture),
+      capable of running DOS programs that require real or protected mode.
     </p>
     <p>
-      DOSBox emulates a 286/386 realmode CPU, Directory FileSystem/XMS/EMS, a SoundBlaster card
-      for excellent sound compatibility with older games... You can "re-live" the good old days
-      with the help of DOSBox Staging, it can run plenty of the old classics that don't run on your
-      new computer!
+      It features built-in DOS-like shell terminal, emulation of several PC variants
+      (IBM PC, IBM PCjr, Tandy 1000), CPUs (286, 386, 486, Pentium I), graphics
+      chipsets (Hercules, CGA, EGA, VGA, SVGA), audio solutions (Sound Blaster,
+      Gravis UltraSound, Disney Sound Source, Tandy Sound System), CD Digital Audio
+      emulation (also with audio encoded as FLAC, Opus, OGG/Vorbis, MP3 or WAV),
+      joystick emulation (supports modern game controllers), serial port emulation,
+      IPX over UDP, GLSL shaders, and more.
```

Comment:
If light-weight HTML is allowed include the XML tags (given the use of the paragraph breaks), it would be nice for this list of lists to render as such (otherwise it's quite a mouthful).

``` xml
It features:<ul>
<li>A built-in DOS-like console</li>
<li>Emulation of several PC variants: IBM PC, IBM PCjr, Tandy 1000), CPUs (286, 386, 486, and Pentium I</li>
<li>Graphics chipsets: Hercules, CGA, EGA, VGA, and SVGA</li>
<li>Audio solutions: PC Speaker, Tandy Sound System, Disney Sound Source, Sound Blaster series, and Gravis UltraSound</li>
<li>CDROM and CD Digital Audio with audio optionally encoded as FLAC, Opus, OGG/Vorbis, MP3 or WAV</li>
<li>Joystick emulation including supports modern game controllers</li>
<li>Serial port emulation including IPX over UDP and Telnet over TCP/IP</li>
<li>Hardware-accelerated video output including integer scaling, OpenGL shaders, and more<li>
</ul>
```

Reply:
It's not html nor xhtml: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-description